test.js

```diff
@@ -569,3 +570,30 @@ if (Promise.prototype.finally) {
 		t.is(result.message, 'called');
 	});
 }
+
+test('cancel method kills the spawned process', t => {
+	const spawned = execa('ls');
+	spawned.catch(() => { }); // To handle the CancelError
+	spawned.cancel();
+	t.true(spawned.killed);
+});
+
+test('invoking cancel method throws CancelError', async t => {
+	const spawned = execa('ls');
+	spawned.cancel();
+	await t.throwsAsync(spawned, CancelError);
+});
+
+test('default CancelError message is "command was canceled"', async t => {
+	const spawned = execa('ls');
+	spawned.cancel();
+	const error = await t.throwsAsync(spawned);
```

Comment:
You can do those two lines in a single one: `await t.throwsAsync(spawned, { message: 'command was canceled' })`

Same thing for the error tests.

See https://github.com/avajs/ava/blob/master/docs/03-assertions.md#throwsasyncthrower-expected-message for more information.